src/nvim/option.c

```diff
@@ -1973,6 +1976,11 @@ static void didset_options2(void)
 
   // Parse default for 'wildmode'.
   check_opt_wim();
+
+  xfree(curbuf->b_p_vsts_array);
```

Comment:
This comes from a different patch. Mention it in the commit message. You can combine multiple vim patches if it is required to pass CI.

Reply:
This PR contains the commit this comes from, so it should be fine? I changed the PR title to indicate all commits that it contains for clarity.

Reply:
I see what you mean, will change it to reflect the actual commit history.